### ISSUE TITLE
Could not GET 'http://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.xml'

### ISSUE THREAD

**Issue by liboLiao** [gradle/gradle#8846]
I want to open this gradle project in intellij idea on windows，So I used the "gradlew idea" command according to your instructions.But found the following problems:

>D:\IntelliJ-IDEAProjects\gradle\gradle-3.0.0>gradlew idea
 ..............
>* What went wrong:
>Could not resolve all dependencies for configuration ':performance:reports'.
> Could not resolve jquery:jquery.min:1.11.0.
  Required by:
      org.gradle:performance:3.0-snapshot-1
   > Could not resolve jquery:jquery.min:1.11.0.
      > Could not get resource 'http://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.xml'.
         > Could not GET 'http://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.xml'.
            > Connection reset

I also found that http://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.xml returned 404.
Also, I am in China but I use a proxy so I can access googleapis.

**Comment by blindpirate** [comment]
I'm pretty sure that's because your don't configure proxy for terminal/IDEA properly. Being able to access googleapi in browser doesn't mean it's well-configured in terminal/IDEA. There's nothing Gradle can help.

**Comment by liboLiao** [comment]
@blindpirate sorry about this. I config the proxy for "cmd" terminal on win 7 and test it:

>D:\IntelliJ-IDEAProjects\gradle\gradle-3.0.0>F:\Chorme_Downloads\curl-7.64.0_3-win64-mingw\curl-7.64.0-win64-mingw\bin\curl.exe -vv http://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.xml
>* Expire in 0 ms for 6 (transfer 0x11d7f0)
>* Uses proxy env variable http_proxy == 'http://127.0.0.1:1080'
>*   Trying 127.0.0.1...
>* TCP_NODELAY set
>* Expire in 200 ms for 4 (transfer 0x11d7f0)
>* Connected to 127.0.0.1 (127.0.0.1) port 1080 (#0)
> GET http://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.xml HTTP/1.1
> Host: ajax.googleapis.com
> User-Agent: curl/7.64.0
> Accept: */*
> Proxy-Connection: Keep-Alive
>
>< HTTP/1.1 404 Not Found
< Access-Control-Allow-Origin: *
< Timing-Allow-Origin: *
< X-Content-Type-Options: nosniff
< Date: Sun, 24 Mar 2019 04:22:30 GMT
< Content-Type: text/html; charset=UTF-8
< Server: sffe
< Content-Length: 1599
< X-XSS-Protection: 1; mode=block
< Connection: close
< Proxy-Connection: keep-alive
<
< !    DOCTYPE html  >
< html lang=en  >
  < meta charset=utf-8>
  < meta name=viewport content="initial-scale=1, minimum-scale=1, width=device-width">
  < title>Error 404 (Not Found)!!1</title>
  < style>
    *{margin:0;padding:0}html,code{font:15px/22px arial,sans-serif}html{background:#fff;color:#222;padding:15px}body{mar
gin:7% auto 0;max-width:390px;min-height:180px;padding:30px 0 15px}* > body{background:url(//www.google.com/images/error
s/robot.png) 100% 5px no-repeat;padding-right:205px}p{margin:11px 0 22px;overflow:hidden}ins{color:#777;text-decoration:
none}a img{border:0}@media screen and (max-width:772px){body{background:none;margin-top:0;max-width:none;padding-right:0
}}#logo{background:url(//www.google.com/images/branding/googlelogo/1x/googlelogo_color_150x54dp.png) no-repeat;margin-le
ft:-5px}@media only screen and (min-resolution:192dpi){#logo{background:url(//www.google.com/images/branding/googlelogo/
2x/googlelogo_color_150x54dp.png) no-repeat 0% 0%/100% 100%;-moz-border-image:url(//www.google.com/images/branding/googl
elogo/2x/googlelogo_color_150x54dp.png) 0}}@media only screen and (-webkit-min-device-pixel-ratio:2){#logo{background:ur
l(//www.google.com/images/branding/googlelogo/2x/googlelogo_color_150x54dp.png) no-repeat;-webkit-background-size:100% 1
00%}}#logo{display:inline-block;height:54px;width:150px}
  < /style>
  < a href=//www.google.com/><span id=logo aria-label=Google></span></a>
  < p> <b>404.</b> < ins>That’s an error.</ins>
  < p>The requested URL < code>/ajax/libs/jquery/1.11.0/jquery.min.xml</code> was not found on this server.  < ins>That’s
all we know.</ins>
>* Closing connection 

The proxy is ok,and then,I used the "gradlew idea" command and that problem still appears:

>* What went wrong:
Could not resolve all dependencies for configuration ':performance:reports'.
> Could not resolve jquery:jquery.min:1.11.0.
  Required by:
      org.gradle:performance:3.0-snapshot-1
   > Could not resolve jquery:jquery.min:1.11.0.
      > Could not get resource 'http://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.xml'.
         > Could not GET 'http://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.xml'.
            > Connect to ajax.googleapis.com:80 [ajax.googleapis.com/172.217.161.138, ajax.googleapis.com/172.217.163.23
4, ajax.googleapis.com/172.217.25.10, ajax.googleapis.com/216.58.221.234, ajax.googleapis.com/216.58.199.106, ajax.googl
eapis.com/216.58.220.202, ajax.googleapis.com/172.217.31.234, ajax.googleapis.com/172.217.24.42, ajax.googleapis.com/216
.58.200.10, ajax.googleapis.com/172.217.161.170, ajax.googleapis.com/216.58.199.10, ajax.googleapis.com/172.217.24.202,
ajax.googleapis.com/216.58.200.74] failed: Connection timed out: connect

so I test the "ajax.googleapis.com:80" with "curl.exe -vv ajax.googleapis.com:80".

>D:\IntelliJ-IDEAProjects\gradle\gradle-3.0.0>F:\Chorme_Downloads\curl-7.64.0_3-win64-mingw\curl-7.64.0-win64-mingw\bin\curl.exe -vv ajax.googleapis.com:80
>* Expire in 0 ms for 6 (transfer 0x58d400)
>* Uses proxy env variable http_proxy == 'http://127.0.0.1:1080'
>*   Trying 127.0.0.1...
>* TCP_NODELAY set
>* Expire in 200 ms for 4 (transfer 0x58d400)
>* Connected to 127.0.0.1 (127.0.0.1) port 1080 (#0)
> GET http://ajax.googleapis.com:80/ HTTP/1.1
> Host: ajax.googleapis.com
> User-Agent: curl/7.64.0
> Accept: */*
> Proxy-Connection: Keep-Alive
>
><  HTTP/1.1 302 Found
<  Location: https://developers.google.com/speed/libraries/devguide
<  Cache-Control: private
<  Content-Type: text/html; charset=UTF-8
<  X-Content-Type-Options: nosniff
<  Date: Sun, 24 Mar 2019 05:35:17 GMT
<  Server: sffe
<  Content-Length: 251
<  X-XSS-Protection: 1; mode=block
<  Connection: close
<  Proxy-Connection: keep-alive
><
< HTML>< HEAD><meta http-equiv="content-type" content="text/html;charset=utf-8">
< TITLE>302 Moved</TITLE></HEAD><BODY>
< H1>302 Moved</H1>
The document has moved
< A HREF="https://developers.google.com/speed/libraries/devguide">here</A>.
< /BODY></HTML>
>* Closing connection 0

I think the resource corresponding to “ajax.googleapis.com:80” has been moved on server.

**Comment by blindpirate** [comment]
```
What went wrong:
Could not resolve all dependencies for configuration ':performance:reports'.
Could not resolve jquery:jquery.min:1.11.0.
Required by:
org.gradle:performance:3.0-snapshot-1
Could not resolve jquery:jquery.min:1.11.0.
> Could not get resource 'http://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.xml'.
> Could not GET 'http://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.xml'.
> Connect to ajax.googleapis.com:80 [ajax.googleapis.com/172.217.161.138, ajax.googleapis.com/172.217.163.23
4, ajax.googleapis.com/172.217.25.10, ajax.googleapis.com/216.58.221.234, ajax.googleapis.com/216.58.199.106, ajax.googl
eapis.com/216.58.220.202, ajax.googleapis.com/172.217.31.234, ajax.googleapis.com/172.217.24.42, ajax.googleapis.com/216
.58.200.10, ajax.googleapis.com/172.217.161.170, ajax.googleapis.com/216.58.199.10, ajax.googleapis.com/172.217.24.202,
ajax.googleapis.com/216.58.200.74] failed: Connection timed out: connect
```

Note the failure: `Connection timed out: connect`, this is clear saying that your JVM is not configured to use proxy. Being able to use curl to access googleapis doesn't mean JVM can access googleapis.

BTW, you shouldn't use `./gradlew idea` to generate idea module configuration, use [this doc](https://github.com/gradle/gradle/blob/master/CONTRIBUTING.md#intellij) instead. Anyway, you still need to resolve the proxy issue. A simple workaround would be removing this line: https://github.com/gradle/gradle/blob/1e4bb4edaaa74cde83ef0063e0b40f34dda1ae2f/subprojects/reporting/reporting.gradle#L24

**Comment by liboLiao** [comment]
you`re right!  Thanks a lot for your help.  fixed with the proxy of JVM.
